app/src/forms/common/models/views/submissionData.js

```diff
@@ -21,6 +21,15 @@ class SubmissionData extends Model {
           query.where('createdAt', '<=', maxDate);
         }
       },
+      filterUpdatedAt(query, minDate, maxDate) {
+        if (minDate && maxDate) {
+          query.whereBetween('updatedAt', [minDate, maxDate]);
+        } else if (minDate) {
+          query.where('updatedAt', '>=', minDate);
+        } else if (maxDate) {
+          query.where('updatedAt', '<=', maxDate);
+        }
+      },
```

Comment:
We need unit tests that cover all the code branches that are added here. You can check coverage by running the unit tests and looking at `app/coverage/lcov-report/index.html`.

Reply:
Done